### DISCUSSION TITLE
Is it possible to add a new instance of a plug-in with the seqcli?

### DISCUSSION THREAD

**Discussion by christian-townsend** [datalust/seq-tickets#1257] [Ideas]
I have a large number of plug-in instances for a specific app to add for different Seq signals. I was wondering if plug-in instances can be added via the seqcli so that I can easily loop over it for my various signals? If it is not currently available, are there plans to implement such functionality?

**Comment by nblumhardt** [comment]
Hi Christian,

Thanks for getting in touch. There's no support currently in seqcli, but something like `appinstance create` would be a welcome addition.

There's already support in _Seq.Api_, in case you happen to be able to automate using C# + the _Seq.Api_ package?

**Reply by christian-townsend** [reply]
Thanks for your response, Nick. Ill have a look at the API. Would be nice to have `appinstance create` added in the future.
